### DISCUSSION TITLE
Pymodbus is now declared a critical project.

### DISCUSSION THREAD

**Discussion by janiversen** [pymodbus-dev/pymodbus#1563] [Announcements]
Today pypi.org decided that pypi is a critical project, which is a sort of acknowledgment that our project is being included in more distributions.

The downside is that maintainers have to change the upload procedure.

Let’s make pymodbus even better and more used.

**Comment by serroba** [comment]
Today pypi.org decided that *pymodbus* is a critical project

**Comment by janiversen** [comment]
Thanks for correcting the typo, but that does not change the fact, that I at least are proud to maintain this project.

**Reply by serroba** [reply]
Yes, absolutely! Congratulations!

**Reply by tracernz** [reply]
You’re all doing a great job! 👏

**Comment by dhoomakethu** [comment]
congrats @janiversen for the efforts you are putting in. Appreciated.
